PubnativeLite/PubnativeLiteDemo/Adapters/AdMobAdapter/HyBidAdMobBannerCustomEvent.m

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright © 2019 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "HyBidAdMobBannerCustomEvent.h"
+#import "HyBidAdMobUtils.h"
+
+@interface HyBidAdMobBannerCustomEvent() <HyBidAdViewDelegate>
+
+@property (nonatomic, strong) HyBidBannerAdView *bannerAdView;
+
+@end
+
+@implementation HyBidAdMobBannerCustomEvent
+
+@synthesize delegate;
+
+- (void)dealloc {
+    self.bannerAdView = nil;
+}
+
+- (void)requestBannerAd:(GADAdSize)adSize
+              parameter:(NSString * _Nullable)serverParameter
+                  label:(NSString * _Nullable)serverLabel
+                request:(nonnull GADCustomEventRequest *)request {
+    if ([HyBidAdMobUtils areExtrasValid:serverParameter]) {
+        if (CGSizeEqualToSize(kGADAdSizeBanner.size, adSize.size)) {
```

Comment:
won't this affect if the banner is full width?

Reply:
No i guess it won't affect since `kGADAdSizeBanner` is like a constant defined for a size of 320x50.